lms/static/styles/components/_LMSGrader.scss

```diff
@@ -1,39 +1,55 @@
 .LMSGrader {
   display: flex;
   flex-direction: column;
-  font-size: 18px;
+  font-size: 16px;
   height: 100%;
   width: 100%;
   position: fixed;
-  color: white;
-  .LMSGrader__course-assignment {
-    h2, h3 {
-      margin-left: 20px;
-    }
-  }
+  color: $grey-6;
 }
 
 .LMSGrader__grading-components {
-  background-color: $grey-7;
-  padding: 0;
+  background-color: white;
```

Comment:
Do we have a variable for this? I feel like we should be consistent in using variables for all colors.